tests/output_preparation.rs

```diff
@@ -372,3 +372,70 @@ async fn output_preparation() -> Result<()> {
 
     common::tear_down(storage_path)
 }
+
+#[ignore]
+#[tokio::test]
+async fn prepare_nft_output_features_update() -> Result<()> {
+    let storage_path = "test-storage/output_preparation";
+    common::setup(storage_path)?;
+
+    let manager = common::make_manager(storage_path, None, None).await?;
+    let accounts = &common::create_accounts_with_funds(&manager, 2).await?;
```

Comment:
```suggestion
    let accounts = &common::create_accounts_with_funds(&manager, 1).await?;
```
You only use one account